### ISSUE TITLE
Modx3 compatibility

### ISSUE THREAD

**Issue by nsuomine** [bezumkin/Tickets#173]
It seems that Tickets use deprecated functions of ModX.

**Comment by shamarin** [comment]
Yes, it's true. With installed Tickets on MODX 3 resources tree became unavailable. In console I get 500 error 
`POST /connectors/index.php?action=Resource/GetNodes&id=web_0&type=MODX\Revolution\modContext`
`PHP Notice:  Undefined index: modUser in /home/n/natasc/buhuslugi/public_html/core/vendor/xpdo/xpdo/src/xPDO/xPDOMap.php on line 44
PHP Notice:  Indirect modification of overloaded element of xPDO\xPDOMap has no effect in /home/n/natasc/buhuslugi/public_html/core/components/tickets/model/tickets/metadata.mysql.php on line 26

`

**Comment by Ibochkarev** [comment]
@shamarin @nsuomine в ближайшее время будет обновление или новая версия под MODX 3
